packages/fxa-content-server/app/scripts/templates/sign_in_password.mustache

```diff
@@ -32,7 +32,7 @@
       {{/isPasswordNeeded}}
 
       <div class="button-row">
-        <button id="submit-btn" type="submit">{{#t}}Sign in{{/t}}</button>
+        <button id="submit-btn" {{^isPasswordNeeded}}class="use-logged-in"{{/isPasswordNeeded}} type="submit">{{#t}}Sign in{{/t}}</button>
```

Comment:
This change was just so I could check for the expected button from the functional tests.